src/car/simple/SimpleCar.hpp

```diff
@@ -25,6 +25,8 @@ class SimpleCar : public Car
      */
     SimpleCar(Control& control);
 
+    virtual ~SimpleCar() = default;
```

Comment:
:warning: **modernize-use-override** :warning:
prefer using 'override' or (rarely) 'final' instead of 'virtual'
```suggestion
    virtual ```
```suggestion
    virtual override ```